cmd/admin-subnet-register.go

```diff
@@ -18,160 +18,20 @@
 package cmd
 
 import (
-	"bufio"
-	"fmt"
-	"os"
-	"strings"
-
-	"github.com/google/uuid"
 	"github.com/minio/cli"
-	"github.com/minio/mc/pkg/probe"
 	"github.com/minio/pkg/console"
 )
 
 var adminSubnetRegisterCmd = cli.Command{
-	Name:         "register",
-	Usage:        "Register the MinIO Cluster with SUBNET",
-	OnUsageError: onUsageError,
-	Action:       mainAdminRegister,
-	Before:       setGlobalsFromContext,
-	Flags:        append(subnetCommonFlags, globalFlags...),
-	CustomHelpTemplate: `NAME:
-  {{.HelpName}} - {{.Usage}}
-
-USAGE:
-  {{.HelpName}} TARGET
-
-FLAGS:
-  {{range .VisibleFlags}}{{.}}
-  {{end}}
-EXAMPLES:
-  1. Register MinIO cluster at alias 'play' on SUBNET, using alias as the cluster name.
-     {{.Prompt}} {{.HelpName}} play
-
-  2. Register MinIO cluster at alias 'play' on SUBNET, using the name "play-cluster".
-     {{.Prompt}} {{.HelpName}} play --name play-cluster
-
-  3. Register MinIO cluster at alias 'play' on SUBNET, using the proxy https://192.168.1.3:3128
-     {{.Prompt}} {{.HelpName}} play --subnet-proxy https://192.168.1.3:3128
-`,
-}
-
-// checkAdminRegisterSyntax - validate arguments passed by a user
-func checkAdminRegisterSyntax(ctx *cli.Context) {
-	if len(ctx.Args()) == 0 || len(ctx.Args()) > 1 {
-		cli.ShowCommandHelpAndExit(ctx, "register", 1) // last argument is exit code
-	}
-}
-
-// ClusterRegistrationReq - JSON payload of the subnet api for cluster registration
-// Contains a registration token created by base64 encoding  of the registration info
-type ClusterRegistrationReq struct {
-	Token string `json:"token"`
-}
-
-// ClusterRegistrationInfo - Information stored in the cluster registration token
-type ClusterRegistrationInfo struct {
-	DeploymentID string      `json:"deployment_id"`
-	ClusterName  string      `json:"cluster_name"`
-	UsedCapacity uint64      `json:"used_capacity"`
-	Info         ClusterInfo `json:"info"`
-}
-
-// ClusterInfo - The "info" sub-node of the cluster registration information struct
-// Intended to be extensible i.e. more fields will be added as and when required
-type ClusterInfo struct {
-	MinioVersion    string `json:"minio_version"`
-	NoOfServerPools int    `json:"no_of_server_pools"`
-	NoOfServers     int    `json:"no_of_servers"`
-	NoOfDrives      int    `json:"no_of_drives"`
-	NoOfBuckets     uint64 `json:"no_of_buckets"`
-	NoOfObjects     uint64 `json:"no_of_objects"`
-	TotalDriveSpace uint64 `json:"total_drive_space"`
-	UsedDriveSpace  uint64 `json:"used_drive_space"`
-}
-
-// SubnetLoginReq - JSON payload of the SUBNET login api
-type SubnetLoginReq struct {
-	Username string `json:"username"`
-	Password string `json:"password"`
-}
-
-// SubnetMFAReq - JSON payload of the SUBNET mfa api
-type SubnetMFAReq struct {
-	Username string `json:"username"`
-	OTP      string `json:"otp"`
-	Token    string `json:"token"`
+	Name:               "register",
```

Comment:
I think we have Hidden flag here, I guess you can use it, can you double check ?

Reply:
Yes, thanks. Done.